### PR TITLE
Fix typo in config-no-entropy.h

### PR DESCRIPTION
**Pull request by jacobschloss**
## Description
Fix typo, reatures -> features

## Status
READY

## Requires Backporting
NO  

## Migrations
NO

### CODE REVIEW

**Comment by RonEld** [comment]
@jacobschloss Thank you for your contribution!

As straight forward this fix is, unfortunately our policy is to not accept contributions, without a Contributor’s Licence Agreement (CLA) signed or authorised by yourself or your employer.

If this is a personal contribution, the easiest way to do this is if you create an mbed account and [accept this click through agreement](https://os.mbed.com/contributor_agreement/). Alternatively, you can find a [slightly different agreement to sign here](https://www.mbed.com/en/about-mbed/contributor-license-agreements/), which can be signed and returned to us, and is applicable if you don't want to create an mbed account or alternatively if this is a corporate contribution.

Thanks for your understanding and again, thanks for the contribution!

**Comment by jacobschloss** [comment]
Thanks, signed pdf CLA is now submitted.

**Comment by RonEld** [comment]
@jacobschloss Thank you for submitting the CLA.

We usually add ChangeLog entries for external contributors, crediting their contributions.
Would you be willing to add a ChangeLog entry, crediting yourself for the fix?

**Comment by jacobschloss** [comment]
Sure, added a line to ChangeLog and rebased.

File: ChangeLog [hunk 1]
```diff
@@ -75,6 +75,7 @@ Changes
      been disabled for lack of a sufficiently recent version of GnuTLS on the CI.
    * Ciphersuites based on 3DES now have the lowest priority by default when
      they are enabled.
+   * Fix typo in config-no-entropy.h submitted by Jacob Schloss in #2479
```
Comment (RonEld) [inline ChangeLog]:
```suggestion
   * Fix typo in config-no-entropy.h submitted by Jacob Schloss in #2479.
```
Note to gatekeeper:
Add '.' at end of sentence

**Comment by RonEld** [comment]
This typo is in `mbedtls-2.7` as well, so this PR will require backporting as well

File: ChangeLog (hunk 1 above)
Comment (RonEld) [inline ChangeLog]: This entry should be in the`Bugfix` section, and not in the `changes` section

Comment (jacobschloss) [reply ChangeLog]: oops, moved entry to Bugfix

**Review by andresag01 — DISMISSED** [review]

**Review by RonEld — DISMISSED** [review]

**Comment by tom-cosgrove-arm** [comment]
Backport: #5713

**Review by tom-daubney-arm — APPROVED** [review]
LGTM (Pending CI completion)

**Comment by tom-daubney-arm** [comment]
CI has failed but I have a feeling this failure is not to do with anything in the PR.

**Comment by tom-cosgrove-arm** [comment]
Agreed, looks like OpenCI gave up on one of the builds because of an update to the PR, but has reported it as an error (the TLS Testing one) and the other has been stuck waiting for resource for a while... all other parts of the CI have passed (not surprisingly, given this is a comment-only change!)

**Pull request merged**